code-name-taran/src/com/spaghettisoft/component/dontgetangry/Board.java

```diff
@@ -0,0 +1,69 @@
+package com.spaghettisoft.component.dontgetangry;
+
+
+public class Board {
+
+    public static final String BLANK = "[ ]";	

```

Comment:
@georgiatanasov147 
Static methods are bad idea. You should try to avoid them. It will lead to bad design and spaghetti code. It also makes the code hard to test and maintain.